includes/admin/tasks/class-wc-payments-task-disputes.php

```diff
@@ -64,7 +64,7 @@ public function __construct() {
 	 * Initialize the task.
 	 */
 	private function init() {
-		$this->disputes_due_within_7d = $this->get_disputes_needing_response_within_days( 7 );
+		$this->disputes_due_within_7d = $this->get_disputes_needing_response_within_days( 30 );
```

Comment:
Note this change may be a mistake – we intentionally only show the task for disputes due within 7 days.

Reply:
Great catch, @Jinksi! I made that change temporarily to test the PR and forgot to change it back 🤦‍♂️ Thanks!